### PR TITLE
[3.5] bpo-19225: Lack of c api exceptions doc

### PR DESCRIPTION
**Pull request by cocoatomo**
backports to 3.5 for #881

### CODE REVIEW

**Comment by cocoatomo** [comment]
Related Sphinx bug report: https://github.com/sphinx-doc/sphinx/issues/3598

File: Doc/c-api/exceptions.rst [hunk 1]
```diff
@@ -735,6 +735,61 @@ All standard Python exceptions are available as global variables whose names are
...
+   single: PyExc_ImportError
+   single: PyExc_IndentationError
+   single: PyExc_IndexError
+   single: PyExc_InterruptedError
+   single: PyExc_IsADirectoryError
+   single: PyExc_KeyError
+   single: PyExc_KeyboardInterrupt
+   single: PyExc_LookupError
+   single: PyExc_MemoryError
+   single: PyExc_ModuleNotFoundError
```
Comment (serhiy-storchaka) [inline Doc/c-api/exceptions.rst]: New in 3.6.

**Review by serhiy-storchaka — CHANGES_REQUESTED** [review]

File: Doc/c-api/exceptions.rst (hunk 1 above)
Comment (cocoatomo) [reply Doc/c-api/exceptions.rst]: Thanks!

**Comment by cocoatomo** [comment]
@serhiy-storchaka Is there additional work needed?

**Review by serhiy-storchaka — APPROVED** [review]

**Comment by serhiy-storchaka** [comment]
All LGTM. Thanks.

**Pull request merged**